lib/ui/compositing/scene_host.h

```diff
@@ -12,12 +12,36 @@
 #include <zircon/types.h>
 
 #include "dart-pkg/zircon/sdk_ext/handle.h"
+#include "dart/runtime/include/dart_api.h"
 #include "flutter/fml/memory/ref_counted.h"
 #include "flutter/fml/task_runner.h"
 #include "flutter/lib/ui/dart_wrapper.h"
 
 namespace flutter {
 
+struct SceneHostBindingKey {
```

Comment:
Can you move all of this and the #include into the .cc file?

Reply:
yup!